doc/user/ospfd.rst

```diff
@@ -1055,6 +1055,255 @@ TI-LFA requires a proper Segment Routing configuration.
+   Algorithm Definition TLV (carried inside the Router-Info external LSA).
+
+.. clicmd:: no flexible-algorithm (128-255) exclude-admin-group (0-4294967295)
+
+   Remove the specified `admin-group` from the list of Exclude-Admin-Groups
+   for the Flexible Algorithm Definition specified by the algorithm-id.
+   This will remove the admin-group from the list of admin-groups advertised in
+   the Exclude-Admin-Group Sub-TLV included in the corresponding Flexible
+   Algorithm Definition TLV (carried inside the Router-Info external LSA).
+
+.. clicmd:: flexible-algorithm (128-255) include-any-admin-group (0-4294967295)
+
+   Add the specified `admin-group` to the list of Include-Any-Admin-Groups
+   for the Flexible Algorithm Definition specified by the algorithm-id.
+   This will add the admin-group to the list of admin-groups advertised in
+   the Include-Any-Admin-Group Sub-TLV included in the corresponding Flexible
+   Algorithm Definition TLV (carried inside the Router-Info external LSA).
+
+.. clicmd:: no flexible-algorithm (128-255) include-any-admin-group (0-4294967295)
+
+   Remove the specified `admin-group` from the list of Include-Any-Admin-Groups
+   for the Flexible Algorithm Definition specified by the algorithm-id.
+   This will remove the admin-group from the list of admin-groups advertised in
+   the Include-Any-Admin-Group Sub-TLV included in the corresponding Flexible
+   Algorithm Definition TLV (carried inside the Router-Info external LSA).
+
+.. clicmd:: flexible-algorithm (128-255) include-all-admin-group (0-4294967295)
+
+   Add the specified `admin-group` to the list of Include-All-Admin-Groups
+   for the Flexible Algorithm Definition specified by the algorithm-id.
+   This will add the admin-group to the list of admin-groups advertised in
+   the Include-All-Admin-Group Sub-TLV included in the corresponding Flexible
+   Algorithm Definition TLV (carried inside the Router-Info external LSA).
+
+.. clicmd:: no flexible-algorithm (128-255) include-all-admin-group (0-4294967295)
+
+   Remove the specified `admin-group` from the list of Include-All-Admin-Groups
+   for the Flexible Algorithm Definition specified by the algorithm-id.
+   This will remove the admin-group from the list of admin-groups advertised in
+   the Include-All-Admin-Group Sub-TLV included in the corresponding Flexible
+   Algorithm Definition TLV (carried inside the Router-Info external LSA).
+
+.. clicmd:: flexible-algorithm (128-255) exclude-srlg (0-4294967295)
+
+   Add the specified `srlg` value to the list of Exclude-SRLGs
+   for the Flexible Algorithm Definition specified by the algorithm-id.
+   This will add the SRLG vslue to the list of SRLGs advertised in
+   the Exclude-SRLG Sub-TLV included in the corresponding Flexible
+   Algorithm Definition TLV (carried inside the Router-Info external LSA).
+
+.. clicmd:: no flexible-algorithm (128-255) exclude-srlg (0-4294967295)
+
```

Comment:
no need to have 2 lines for both no form and positive form

Reply:
Thanks for the suggestion. Will change it accordingly.